src/Context.php

```diff
@@ -8,7 +8,7 @@
  * Context carries timeout and other request-scoped values across API boundaries
  * and between processes.
  *
- * Timeout is also used to derive a deadline, that is, a pofloat time time after
+ * Timeout is also used to derive a deadline, that is, a point in time after
  * which the context is no longer valid.
  */
 final class Context
```

Comment:
It's not so much that the "context is invalid", but that things using this context are cancelled. Not sure exactly how to word it, but maybe:

> Timeout specifies the maximum amount of time (in seconds) that an operation may take before it is cancelled.